### ISSUE TITLE
QUASI_RANDOM_SEARCH no work

### ISSUE THREAD

**Issue by BYTERHIT** [google/vizier#541]
I try to quasi random search points in the range [0.0001, 0.1]. I try for 25 trials but I found the points are in the range [0.01,0.013].  the server and the client code is as the appendix.
[code.zip](https://github.com/google/vizier/files/10892037/code.zip)

**Comment by xingyousong** [comment]
Hi @BYTERHIT, I took a look at your code - just making sure, for the range you're describing, are you using LOG or REVERSE_LOG scale? 

As as FYI, when using LOG or REVERSE_LOG, the algorithm will apply a log-scale to transform your range `[a,b]` down to [0,1]`, uniformly sample points in this underlying `[0,1]`, and then convert back to `[a,b]`. 

Note also that Quasi-Random-Search is not like regular random search; the points sampled in the underlying `[0,1]` space will be fairly spread out; see https://en.wikipedia.org/wiki/Halton_sequence, which may be the cause of your observed behavior in `[0.01, 0.013]`. 

To double check this is a matter of the algorithm, can you try just regular `RANDOM_SEARCH` and see what the sampled points are?

**Comment by BYTERHIT** [comment]
Hi @xingyousong I have checked that when using RANDOM_SEARCH, I get the normal results, the points are in the range [0.0001,0.1]. As follows: 
![WechatIMG30](https://user-images.githubusercontent.com/6222897/223000874-ed1e10a9-fb1b-482d-9927-c9d9dc15be93.jpeg)
And I did not use LOG or REVERSE_LOG scale, instead, I use LINEAR scale. The QUASI_RANDOM_SEARCH results as follows:

![quasi_random](https://user-images.githubusercontent.com/6222897/223001446-88707ba8-ccfc-4b7d-9acc-99a2059ae657.jpeg)
Also I posted the config file for vizier client parameters and the python file to generate the figure above.
[config&plot.zip](https://github.com/google/vizier/files/10893655/config.plot.zip)

Since the sample points are in a small fraction of the target range. I think it is abnormal. Using Halton Sequence should not get the results?
